### ISSUE TITLE
provide a mechanism for serving resources data from a static file

### ISSUE THREAD

**Issue by naomiblack** [angular/angular.io#1677]
We have the resources data in a firebase db so it's easy to update, but this is problematic for angular.cn, which can't connect back to firebase for data. 

Can we come up with a mechanism for automating a flat export of the data from firebase to somewhere that angular.cn can easily pick it up? Maybe a push to github?

Needs some thought.

@rexebin @asnowwolf @ericjim @juleskremer @StephenFluin

**Comment by StephenFluin** [comment]
Firebase works equally well as a flat .json file in read-only use cases. Perhaps we could just proxy it? Then we have to find a spot to host the proxy.

`<?php print file_get_contents("https://angularresources.firebaseio.com/.json");`

**Comment by rexebin** [comment]
Currently, we simply grab the query result from https://angularresources.firebaseio.com and serialize it to a Json object in the ResourceCtrl, replacing the query with a Json object, it works well but won't update automatically. Even if the Firebase works equally well with proxies, it is not easy to translate the content. Therefore, convert the result to static content is our best bet for now. Furthermore, some of the the links in resources may not be accessible or relevant in China. After we establish our community for a little while longer, we would have good local resources which are more relevant to the region.

**Comment by rexebin** [comment]
Also, we removed the content in resources/index.ejs and instead we use resources/index.jade for the template. We found that EJS template is not stable i.e it is unpredictable, maybe due to the fact that we are not familiar with it, thus in our opinion unless we gain significant advantages from using EJS template, there is little point introducing it into the mix.

**Comment by ericjim** [comment]
@rexebin I agree with your assessment for replacing the query with a json object. We would probably have to update the file every time we redeploy. I used .ejs because it looks more like html, and it looked less confusing to me when mixing it with angular. (versus angular with jade). We can use jade for consistency.

**Comment by naomiblack** [comment]
+1 for using JSON for the localized site
